platform/darwin/src/MGLStyleValue_Private.h

```diff
@@ -5,6 +5,10 @@
 #import "NSValue+MGLStyleAttributeAdditions.h"
 #import "MGLTypes.h"
 
+#include "darwin_conversion.hpp"
```

Comment:
The convention for our predominantly Objective-C targets is to use .h for Objective-C++ headers. We also use MGLCamelCase for these headers.